src/common/mfu_io.c

```diff
@@ -646,23 +596,16 @@ int daos_utimensat(int dirfd, const char* pathname, const struct timespec times[
     } else {
         /* Lookup the object */
         dfs_obj_t* obj;
-        rc = dfs_lookup_rel(mfu_file->dfs, parent, name, O_RDWR, &obj, NULL, NULL);
+	int lookup_flags = O_RDWR;
+	if (flags & AT_SYMLINK_NOFOLLOW) {
+            lookup_flags |= O_NOFOLLOW;
+        }
+        rc = dfs_lookup_rel(mfu_file->dfs, parent, name, lookup_flags, &obj, NULL, NULL);
```

Comment:
I'm not familiar with DAOS, but if this call fails with an error, is it still safe to call ``dfs_release(obj)`` or would that cause a problem?

Reply:
If dfs_lookup_rel fails, then dfs_release would return an error. 
But it does look like we're only calling dfs_release when lookup did NOT fail, right?

Reply:
Ah, yes.  Sorry, I skimmed to quickly and missed the else branch here.

Reply:
👍 No worries!